app-frontend/src/app/components/pages/project/layer/aoi/index.html

```diff
@@ -1 +1,69 @@
-<div>AOI</div>
+<rf-aoi-draw-toolbar
```

Comment:
![image](https://user-images.githubusercontent.com/2442245/53575565-d429d300-3b3f-11e9-8f00-5776ef96636c.png)

This toolbar is preventing clicks from going through to the map in this area.